tests/GameSync.UI.Unit.Tests/GameSync.ViewModels.Unit.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'm not sure if we want multiple unittest projects

Reply:
I think it could be a good option to mirror 'architecture' in unit tests from the application. We have separate project for models and for viewModels - so I created separate projects for unit tests for those units.